app/assets/javascripts/components/student/student_comments.es6.jsx

```diff
@@ -69,46 +52,40 @@ class StudentComments extends Component {
     };
   }
 
-
   // --------------------------------------------------
-  // Lifecycle
+  // Handlers
   // --------------------------------------------------
-  componentWillMount() {
-    this.setState(StudentCommentsStore.getState());
-  }
-
-  componentDidMount() {
-    StudentCommentsStore.listen(this._listener);
-    StudentCommentsActions.fetchStudentComments(this.props.id);
-  }
-
-  componentWillUnmount() {
-    StudentCommentsStore.unlisten(this._listener);
+  handleClick(event) {
+    StudentActions.storeOverlay(
+      true,
+      TypeConstants.overlay.type.create,
+      TypeConstants.overlay.target.comment
+    );
```

Comment:
When arrow syntax definitions are too long @unzunz, we can just abstract the functionality into a separate function!